core/src/commonMain/kotlin/org/jetbrains/litmuskt/LitmusOutcomeStats.kt

```diff
@@ -22,6 +22,8 @@ data class LitmusOutcomeSpec(
         in forbidden -> LitmusOutcomeType.FORBIDDEN
         else -> default
     }
+
+    fun all() = accepted + interesting + forbidden
```

Comment:
Let's make `all` a property (similar to `accepted`, `interesting` and `forbidden`). 

Because all of them are immutable, you can actually calculate `all` one time in the constructor.

Reply:
:ok_hand: